specification/agricultureplatform/AgriculturePlatform.DataPlane/examples/2024-11-01-preview/Catalogs_CreateOrUpdate.json

```diff
@@ -0,0 +1,83 @@
+{
+  "title": "Catalogs_CreateOrUpdate",
+  "operationId": "CatalogsOperations_CreateOrUpdate",
+  "parameters": {
+    "api-version": "2024-11-01-preview",
+    "catalogId": "CATALOG123",
+    "catalog": {
```

Comment:
I think CI is complaining because this isn't named `resource`. Please look into fixing CI failures and reach out in the TypeSpec Discussion channel as needed